pkg/volume/hostpath/host_path_test.go

```diff
@@ -196,7 +200,14 @@ func TestProvisioner(t *testing.T) {
 		t.Errorf("Expected reclaim policy %+v but got %+v", v1.PersistentVolumeReclaimDelete, pv.Spec.PersistentVolumeReclaimPolicy)
 	}
 
+	pvDir := filepath.Dir(pv.Spec.HostPath.Path)
+	// avoid to delete unexpected dir when provisioner changes its hostpath default path.
```

Comment:
Can the default path actually be changed? From the code it looks completely hardcoded.

Reply:
What are the chances that someone is running a real local cluster with provisioned hostpath volumes and running kubernetes unit tests at the same time?

It seems safer if we can make the provision directory configurable so that the tests can provision volumes in a dedicated place.

Reply:
> What are the chances that someone is running a real local cluster with provisioned hostpath volumes and running kubernetes unit tests at the same time?
> 
> It seems safer if we can make the provision directory configurable so that the tests can provision volumes in a dedicated place.

Yes , there are some hard codes in the host path provisioner, I also think it is better to make it configurable